modules/claims_api/app/controllers/concerns/claims_api/disability_compensation_validations.rb

```diff
@@ -23,6 +23,8 @@ def validate_form_526_submission_values!
       validate_form_526_veteran_homelessness!
       # ensure 'militaryRetiredPay.receiving' and 'militaryRetiredPay.willReceiveInFuture' are not same non-null values
       validate_form_526_service_pay!
+      # checks unit phone
+      validate_form_526_reserves_unit_phone!
```

Comment:
this is a validation file so I"m not sure I align with doing this removal here. I think we should look at where it's mapped in `auto_established_claim#to_internal`. That's more in line with the mapping / data structuring.